rustls/src/msgs/deframer.rs

```diff
@@ -155,19 +155,21 @@ impl MessageDeframer {
 
             // If it's not a handshake message, just return it -- no joining necessary.
             if msg.typ != ContentType::Handshake {
+                let message = msg.into_owned();
```

Comment:
I'm not sure I spot how these `MessageDeframer` changes map to the `Decrypted` update. Possibly a candidate to move into a different commit?

Reply:
That's because this `msg` value comes from a `Decrypted` value, which has a `BorrowedPlainMessage` now instead of a `PlainMessage`